tensorflow/compiler/xla/service/gpu/backend_configs.proto

```diff
@@ -88,3 +88,22 @@ message BitcastBackendConfig {
   LayoutProto source_layout = 1;
   LayoutProto result_layout = 2;
 }
+
+// Backend config for a fused Multi-Headed Attention (fMHA) that runs through
+// cudnn.
+message CudnnfMHABackendConfig {
```

Comment:
Maybe just MHABackendConfig, given that names in this file are generally library-agnostic?

Reply:
Done.